src/relentless/simulate/hoomd.py

```diff
@@ -1263,11 +1257,22 @@ def reset(self):
             if self.rdf_params is not None:
                 self._rdf = collections.PairMatrix(self.types)
                 for i, j in self._rdf:
-                    self._rdf[i, j] = freud.density.RDF(
-                        bins=self.rdf_params["bins"],
-                        r_max=self.rdf_params["stop"],
-                        normalize=(i == j),
-                    )
+                    if packaging.version.parse(freud.__version__).major == 2:
```

Comment:
Two things here:
1. I always put the higher version first in an if-elif so that it's easier to drop the lower versions later on.
2. There is a lot of duplication here because we actually only need to change the `normalize` vs. `normalization` keyword. Can we use a dict to simplify this like:
```
freud_rdf_args = dict(
    bins=self.rdf_params["bins"],
    r_max=self.rdf_params["stop"],
)
if _freud_version.major == 3:
    freud_rdf_args.update(normalization_mode="finite_size" if i ==j else "exact")
elif _freud_version.major == 2:
    freud_rdf_args.update(normalize=(i==j))
self._rdf[i, j] = freud.density.RDF(**freud_rdf_args)
```